
- biarch enabled binutils for kfreebsd-amd64
- hack elfosabi - same as in binutils_elfosabi.diff 


--- binutils-2.16.1cvs20060117/debian/rules	2006-04-10 16:10:19.000000000 +0200
+++ binutils-2.16.1cvs20060117/debian/rules	2006-04-10 15:40:43.000000000 +0200
@@ -69,6 +69,9 @@
 ifeq ($(DEB_HOST_ARCH),i386)
 	CONFARGS += --enable-targets=x86_64-linux-gnu
 endif
+ifeq ($(DEB_HOST_ARCH),kfreebsd-amd64)
+	CONFARGS += --enable-targets=i486-kfreebsd-gnu x86_64-kfreebsd-gnu
+endif
 ifeq ($(DEB_HOST_ARCH),mips)
        CONFARGS += --enable-targets=mips64-linux-gnu
 endif

only in patch2:
unchanged:
--- binutils-2.16.1cvs20060117.orig/bfd/elf.c
+++ binutils-2.16.1cvs20060117/bfd/elf.c
@@ -4802,6 +4802,8 @@
     bfd_big_endian (abfd) ? ELFDATA2MSB : ELFDATA2LSB;
   i_ehdrp->e_ident[EI_VERSION] = bed->s->ev_current;
 
+  i_ehdrp->e_ident[EI_OSABI] = ELFOSABI_FREEBSD;
+
   if ((abfd->flags & DYNAMIC) != 0)
     i_ehdrp->e_type = ET_DYN;
   else if ((abfd->flags & EXEC_P) != 0)
